app/models/vim_performance_state.rb

```diff
@@ -3,7 +3,8 @@ class VimPerformanceState < ApplicationRecord
 
   belongs_to :resource, :polymorphic => true
 
-  ASSOCIATIONS = [:vms, :hosts, :ems_clusters, :ext_management_systems, :storages, :container_nodes, :container_groups]
+  ASSOCIATIONS = [:vms, :hosts, :ems_clusters, :ext_management_systems, :storages, :container_nodes, :container_groups,
```

Comment:
I wasnt sure it wasnt used for anything other than `ContainerProject`s. in `rollup.rb` it refers specifically ContainerProjects so the other wasnt changed. 
If there isnt any other entity that needs to rollup ContainerGroup metrics then we can get rid of the old one.